src/routes/chat/components/chatInput/ChatInput.js

```diff
@@ -1,30 +1,62 @@
-import React, { memo } from 'react';
+import React, { memo, useEffect } from 'react';
+import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import useStyles from 'isomorphic-style-loader/useStyles';
+import property from 'lodash/property';
+import isEmpty from 'lodash/isEmpty';
 
 import s from './ChatInput.css';
 
-function ChatInput({ handleChange, userInput, sendUserMessage }) {
+function ChatInput({
+  isEditMessage,
+  handleChange,
+  updateUserMessage,
+  userInput,
+  sendUserMessage,
+  setUserInput,
+}) {
   useStyles(s);
+  const editMessageText = useSelector(property('chat.editMessageText'));
+  useEffect(() => {
+    if (
+      // isEmpty does not work as expected
+      isEditMessage === true &&
+      (userInput === '' || isEmpty(userInput) || undefined)
```

Comment:
In this place I talking about "isEmpty(isEditMessage)"  instead "isEditMessage === true". This part of code  responds for past edit message text in input.  When I used "isEmpty(isEditMessage)" it would`t happend.

Reply:
userInput === ''"  change to size(userInput) === 0

Reply:
You'll explain in a call